datacenterlight/views.py

```diff
@@ -428,14 +429,15 @@ def get_context_data(self, **kwargs):
         context = super(PaymentOrderView, self).get_context_data(**kwargs)
         context.update({
             'stripe_key': settings.STRIPE_API_PUBLIC_KEY,
-            'site_url': reverse('datacenterlight:index')
+            'site_url': reverse('datacenterlight:index'),
+            'login_form': HostingUserLoginForm()
         })
         return context
 
     @cache_control(no_cache=True, must_revalidate=True, no_store=True)
     def get(self, request, *args, **kwargs):
-        if 'specs' not in request.session or 'user' not in request.session:
-            return HttpResponseRedirect(reverse('datacenterlight:index'))
+        # if 'specs' not in request.session or 'user' not in request.session:
```

Comment:
I wrote this code to prevent accidental back button press after the user having reached the "Thank you for your order" page. Not sure if this is still relevant but we should make sure that case is properly handled.